src/MatrixTypes.f90

```diff
@@ -912,28 +872,47 @@ SUBROUTINE matvec_DistrBandedMatrixType(thisMatrix,x,y,t,ul,d,incx,a,b)
 !> @param req The array of active requests for data
 !> @param idxReq The array of requests for indices
 !>
-SUBROUTINE pop_recv(acc,valBuf,idxBuf,ctBuf,idx,req,idxReq)
+SUBROUTINE pop_recv(acc,valBuf,thisMat,ctBuf,idx,req,f)
```

Comment:
Need to update comment block for change in parameters

Reply:
This was fixed in the last commit. However, there is a type so I will fix this and push up, which will restart the pipeline and hopefully fix the 'stuck' pipeline status.